CMakeLists.txt

```diff
@@ -250,7 +248,7 @@ add_swift_library(Foundation
                     -L${install_dir}/usr/lib
                     -lCoreFoundation
                     ${CURL_LIBRARIES}
-                    ${ICU_UC_LIBRARIES} ${ICU_I18N_LIBRARIES}
+                    ${ICU_UC_LIBRARY} ${ICU_I18N_LIBRARY}
                     ${LIBXML2_LIBRARIES}
```

Comment:
@compnerd This change seems to have broken it and also in the `set(Foundation_INTERFACE_LIBRARIES` below.